style.css

```diff
@@ -0,0 +1,24 @@
+*{
+ margin: 0;
+ padding: 0;
+ box-sizing: border-box;
+}
+#header{
+ /*position: fixed;*/
```

Comment:
Please do not leave unused code as a comment in your code. It will hell to make your code clean.